tools/roxvet/analyzers/declarationcommentmismatch/analyzer.go

```diff
@@ -0,0 +1,104 @@
+package declarationcommentmismatch
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"strings"
+	"unicode"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+const doc = `check that there are no functions, interfaces or structs preceded by descriptive comments of mismatching capitalization`
+
+// Analyzer is the analyzer.
```

Comment:
![6ezxn6](https://user-images.githubusercontent.com/114479/166888202-d2e9d1cb-96a6-421c-947c-81ab55fc39fc.jpg)

Reply:
Blame all the other linters with the exact same comment that I copied 😆

Reply:
Please be the good guy and write something informative - maybe it will be so good that others will copy it from you?

Reply:
I mean `Analyzer` is a `analysis.Analyzer` from the `analysis` package of the Go standard lib. It analyzes code, doesn't the name itself define it plenty? I could write `Analyser is responsible for analyzing code` and your meme still holds true.

Reply:
> Analyzer is a analysis.Analyzer from the analysis package of the Go standard lib. [It analyzes code]

I mean, I would love to have this as the code comment instead of the current one :D
I immediately know way more about it than reading the current comment.

Reply:
Sure lol

Reply:
A good comment would tell us why the thing is here (and optionally what the thing does). So far, I know what it does, but I can only suspect why it is here as a global variable. Guess: I think that each package must have an exported object called Analyzer. Is this correct?

Reply:
I think that is correct because all other linters have one too.